### PR TITLE
Vd photogram pundit

### PR DESCRIPTION
**Pull request by viviancreates**

### CODE REVIEW

File: app/policies/comment_policy.rb
Comment (armstrca) [inline app/policies/comment_policy.rb:1]: Edit/delete icons are still showing up for comments that current_user does not own. Clicking on edit prompts a flash error that the user isn't authorized, which is correct, but the icon shouldn't be visible anyway. 

Clicking on delete does not work, which is correct, but causes this error:
Screenshot 2024-11-05 100224

File: app/policies/user_policy.rb
Comment (armstrca) [inline app/policies/user_policy.rb:1]: Receive this error when trying to go to `root/:user/discover`
Screenshot 2024-11-05 101955

File: app/policies/photo_policy.rb
Comment (armstrca) [inline app/policies/photo_policy.rb:1]: Receive this error when going to `root/:user/liked`

Screenshot 2024-11-05 102203

**Review by armstrca — CHANGES_REQUESTED** [review]
Ideally, users who don't follow a private user should be able to see the private user's profile information without seeing their photos. The assignment gets this wrong despite matchthetarget showing a correct example. I won't grade on that, but it would be good exercise to figure the solution out for yourself.

File: app/controllers/users_controller.rb
Comment (AnnaVi11arrea1) [inline app/controllers/users_controller.rb:1]: Screenshot 2024-11-08 152222
You need to define a liked method in your users controller. :)

Comment (AnnaVi11arrea1) [inline app/controllers/users_controller.rb:1]: Screenshot 2024-11-08 152502
Also need to define a discover method in users controller.

**Review by AnnaVi11arrea1 — CHANGES_REQUESTED** [review]
Need to just define some methods in your users controller. :)

**Review by armstrca — CHANGES_REQUESTED** [review]
The `delete` button doesn't seem to work for comments, but otherwise this looks good.

**Pull request merged**